challenges/objects-arrays.js

```diff
@@ -1,127 +1,188 @@
-// ==== Objects ====
+// // ==== Objects ====
+
+// /* 
+//   Given the following information about dinosaurs, create 3 objects: 
+//   Use this pattern to create your objects: 
+//   object name, diet, weight, length, period
+// */
+
+
+// // const Dinosaur = {
+// //   name: "tryannosaurus",
+// //   diet: "carnivorous",
+// //   weight:  "7000kg",
+// //   length: "12m", 
+// //   period: "Late Cretaceous",
+// //   sound: function(roar){
+// //   console.log("${this.name} makes a loud ${roar}");
+// //   }
+// // }
+// // Dinosaur.sound("RAWERSRARARWERSARARARRRR!");
+// // console.log(Dinosaur);
 
-/* 
-  Given the following information about dinosaurs, create 3 objects: 
-  Use this pattern to create your objects: 
-  object name, diet, weight, length, period
-*/
+// // const Dinosaur2 = {
+// //   name: "stegosaurus",
+// //   diet: "herbivorous",
+// //   weight: "2000kg",
+// //   length: "9m", 
+// //   period: "Late Jurassic"
+// // }
+// // console.log(Dinosaur2);
 
-// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
+// // const Dinosaur3 = {
+// //   name: "velociraptor",
+// //   diet: "carnivorous",
+// //   weight: "15kg",
+// //   length: "1.8m", 
+// //   period: "Late Cretaceous"
+// // }
+// // console.log(Dinosaur3);
 
-// stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
+// // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
 
-// velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
+// // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
 
-// Using your dinosaur objects, log answers to these questions:
+// // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
 
-// How much did tyrannosaurus weigh?
-console.log();
+// // Using your dinosaur objects, log answers to these questions:
 
-// What was the diet of a velociraptor?
-console.log();
+// // How much did tyrannosaurus weigh?
+// // console.log(Dinosaur.weight);
 
-// How long was a stegosaurus?
-console.log();
+// // // What was the diet of a velociraptor?
+// // console.log(Dinosaur3.diet);
 
-// What time period did tyrannosaurus live in?
-console.log();
+// // // How long was a stegosaurus?
+// // console.log(Dinosaur2.length);
 
+// // // What time period did tyrannosaurus live in?
+// // console.log(Dinosaur.period);
```

Comment:
You should name your objects with their name. So instead of `Dinasour2.length` you'll have `stegosaurus.length`.